_posts/en/newsletters/2020-09-02-newsletter.md

```diff
@@ -0,0 +1,243 @@
+conversations and notable changes to popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Witness asymetric payment channels:** Lloyd Fournier [posted][witness asymmetric payment channels] to
+  the Lightning-Dev mailing list this week with a request for feedback
+  on a proposed change to the way LN commitment transactions are
+  created.  Currently, in-channel payments are made by constructing two
+  transactions that conflict with each other, meaning Bitcoin's
+  consensus rules will only allow one of the transactions to be included
+  in the block chain.  Each of these transactions includes the same two
+  outputs with the same two amounts---one paying Alice and one paying
+  Bob---but with slightly different scripts.
+
+  The script for Alice allows her to unilaterally close the channel, but
+  it gives Bob a chance to spend her funds if he knows a secret that
+  Alice previously created.  Similarly, if Bob unilaterally closes the
+  channel, Alice gets a chance to spend his funds using a secret he
+  created.  This ability to give away a secret in order to allow your
+  counterparty to spend your money makes it possible to effective
+  *revoke* old channel states so that the latest unrevoked channel state
+  can be safely published onchain when it comes time to close the
+  channel.
+
+  What Fournier proposed this week was only creating a single commitment
+  transaction (but still with two outputs) that would be shared between
+  Alice and Bob.  When creating the transaction, Alice creates two
+  private keys she'll need to spend her output and gives Bob the
+  corresponding public keys.  Bob then gives Alice a [signature
+  adaptor][topic adaptor signatures] for his half of the 2-of-2 multisig spend in the commitement
+  transaction; the adaptor can only be transformed into Bob's signature
+  if that signature will reveal one of Alice's private keys to Bob.  The
+  other private key Alice created is the revocation private key, which
+  she'll give to Bob if she wants to revoke this channel state in favor
+  of a later state.  Similarly, Bob generates his own keys and receives
+  a signature adaptor from Alice.
+
+  In this way, even though the commitment transaction is the same for
+  both parties, there will be a difference in the signatures (called
+  *witnesses*) each of them uses to unilaterally close the channel.
+  The difference in the signatures will give the party who didn't close
+  the channel a chance to create and broadcast a penalty transaction if
+  the channel was closed in an old state.  Otherwise, if the channel was
+  closed in its latest state, both parties get their money, just like in
+  current LN.
+
+  The proposal also suggests using basically the same scheme to
+  transition from Hash Time Lock Contracts (HTLCs) to Point Time Lock
+  Contracts (PTLCs), which is already a goal several LN contributors
```

Comment:
Could link PTLCs to https://bitcoinops.org/en/newsletters/2020/04/08/#work-on-ptlcs-for-ln-using-simplified-ecdsa-adaptor-signatures for more info

Reply:
Nice!  I forgot we had a decent description of PTLCs.